ci-operator/step-registry/openshift-qe/network-policy/openshift-qe-network-policy-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
+pushd e2e-benchmarking/workloads/kube-burner
+
+export JOB_TIMEOUT=${JOB_TIMEOUT:=21600}
+
+current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/worker --output jsonpath="{.items[?(@.status.conditions[-1].type=='Ready')].status.conditions[-1].type}" | wc -w | xargs)
+
+export WORKLOAD=${WORKLOAD:=networkpolicy-case1}
```

Comment:
I don't think you need this since you set default in the ref file

Reply:
@paigerube14 Thank you for review the code, the script will threw below error if removing WORKLOAD=${WORKLOAD:=networkpolicy-case1}
./openshift-qe-network-policy-commands.sh: line 26: WORKLOAD: unbound variable